### PR TITLE
M3-880, M3-1703: add StackScript detail page with StackScript component

### PR DESCRIPTION
**Pull request by rulikrulit**
- Updates breadcrumbs to make linkText optional
- Adds ScriptCode component to display script
- Add StackScript component to display StackScript details
- Add StackScript Detail page

### CODE REVIEW

**Comment by rulikrulit** [comment]
@WilkinsKa1 @alioso can you check the styling here?

File: src/components/ScriptCode/ScriptCode.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    const {
+      classes,
+      script,
+    } = this.props;
+
+    return (
+      <div className={classes.container}>
+        <table className={classes.table}>
+          {script.split('\n').map((line, counter) => (
+            <tr className={classes.row}>
```
Comment (martinmckenna) [inline src/components/ScriptCode/ScriptCode.tsx]: missing a `key` prop

Comment (martinmckenna) [reply src/components/ScriptCode/ScriptCode.tsx]: Also console warning `Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody> to your code to match the DOM tree generated by the browser.`

File: src/services/images/images.ts [hunk 2]
```diff
@@ -17,6 +17,20 @@ export const getImage = (imageId: string) =>
   )
   .then(response => response.data);
 
+/**
+ * Get information about a multiple Images.
+ *
+ * @param imageIds { string[] } IDs of the Images to look up.
+ */
+export const getImagesByIds = (imageIds: string[]) => {
```
Comment (rjstires) [inline src/services/images/images.ts]: See src/utilities/getAll.ts

File: src/components/StackScript/StackScript.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        </Typography>
+        <Typography variant="h3" className={classes.author}>
+          by&nbsp;
+          <ExternalLink text={username} link={`https://www.linode.com/stackscripts/profile/${username}`} />
+        </Typography>
+        <Typography variant="body2" className={classes.deployments}>
+          {deployments_total} deployments &bull; {deployments_active} still active &bull; last rev. <DateTimeDisplay value={updated} humanizeCutoff={"never"} />
+        </Typography>
+        <div className={classes.description}>
+          <Typography variant="body2">
```
Comment (WilkinsKa1) [inline src/components/StackScript/StackScript.tsx]: Since description isn't a required field, can we hide this if it doesn't exist?

**Pull request merged**